manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -310,7 +310,7 @@ protected function initializeContainer(): void
         $container->set('contao_manager.plugin_loader', $this->getPluginLoader());
 
         // Set the JWT manager only if the debug mode has not been configured in env variables
-        if (!isset($_SERVER['APP_DEBUG']) && !isset($_SERVER['SYMFONY_DEBUG'])) {
+        if (!isset($_SERVER['APP_ENV']) && !isset($_SERVER['SYMFONY_ENV'])) {
```

Comment:
Why did you change this although the comment says to check the debug mode?

Reply:
`APP_DEBUG` was the initial approach to switching the environment. We've changed this because `APP_ENV` is what Symfony primarily listens to, but forgot to change this accordingly.